Assets/Fungus/Scripts/Editor/FlowchartWindow.cs

```diff
@@ -8,11 +8,94 @@
 using System.Linq;
 using System.Collections.Generic;
 using System.Reflection;
+using Object = UnityEngine.Object;
 
 namespace Fungus.EditorUtils
 {
     public class FlowchartWindow : EditorWindow
     {
+        protected class ClipboardObject
+        {
+            internal SerializedObject serializedObject;
+            internal Type type;
+
+            internal ClipboardObject(Object obj)
+            {
+                serializedObject = new SerializedObject(obj);
+                type = obj.GetType();
+            }
+        }
+
+        protected class BlockCopy
+        {
+            private SerializedObject block = null;
+            private List<ClipboardObject> commands = new List<ClipboardObject>();
+            private ClipboardObject eventHandler = null;
+
+            internal BlockCopy(Block block)
```

Comment:
I think it's the most restricted scope while still being accessible elsewhere in the outer class? It could just as well be public; wouldn't make any actual difference. Want me to change it?